estimagic/dashboard/monitoring_app.py

```diff
@@ -91,11 +95,26 @@ def _get_group_to_params_from_database(database):
         return_type="dict_of_lists",
     )
     start_params = optimization_problem["params"][0]
-    group_to_params = _map_groups_to_params(start_params)
-    return start_params, group_to_params
+    start_params["id"] = _create_id_column(start_params)
+    group_to_param_ids = _map_groups_to_param_ids(start_params)
+    return start_params, group_to_param_ids
+
+
+def _create_id_column(df):
```

Comment:
If the ID is only used internally, why not just use integers (if necessary coded as string, i.e. `"0", "1", ...` as IDs?

Reply:
I like that solution because then the identifiers give the location in the parameter array. I also set the id to `'None'` for parameters that are not plotted. (0081c97)